docs/ref/django-admin.txt

```diff
@@ -431,6 +431,17 @@ originally generated.
 
 Specifies a single app to look for fixtures in rather than looking in all apps.
 
+.. django-admin-option:: --exclude EXCLUDE, -e EXCLUDE
+
```

Comment:
For new classes, attributes, options, etc. that have a header like this, you can use `.. versionadded:: 1.10` here instead of `versionchanged`.